.github/workflows/docker-c-build-test.yml

```diff
@@ -55,8 +55,8 @@ jobs:
       - name: Extract C
         run: |
           export HOME=/home/user
-          ./c.sh --no-glue
-
+          ./c.sh --config cg.yaml --mlkem768 --no-unrolling --out c --clean --no-glue
+          cp c/headeronly/eurydice_glue.h c/
```

Comment:
It would be nice if `c.sh` would do this as well such that we can locally run what's run on ci.